site/source/_posts/2020-01-13-announcing-embark-5.md

```diff
@@ -0,0 +1,165 @@
+title: Introducing Embark 5
+author: pascal_precht
+summary: "About half a year after our last stable release, we've now published Embark version 5 with lots of features, improvements and fixes. Read on for more information!"
+categories:
+  - announcements
+layout: blog-post
+---
+
+If you've been following the development of Embark you're probably aware that we regularly put out alpha and beta releases for upcoming major or feature versions of Embark. In the past ~6 months since the release of Embark 4, we've published 10 alpha releases and one beta release for Embark 5 and today we're happy to announce the Embark 5 stable release!
+
+In this post we'll be looking at some of the main changes and features to get up and running with v5.
+
+Let's first talk about the few breaking changes we've introduced to improve the overall developer experience. It's worth noting that we try to keep breaking changes at a minimum and if it's indeed unavoidable, we put lots of effort into keeping the necessary changes as small as possible.
+
+## NodeJS support
+
+Due to some modules and packages that Embark depends on, we can't upgrade to Node's latest stable version (13.x) or latest LTS version (12.x). Embark runs with any node version `>= 10.17.0` and `<12.0.0`. This also includes to have at least `npm v6.11.3`, which comes with node `10.17.0`.
+
+## New Smart Contract configuration API
+
+Embark's Smart Contract configuration has been very declarative from the very first day on. Configuring different deployment options and settings for various scenarios is a very descriptive process when using Embark. However, we felt there was still room for improvement. Especially because Embark handles not only Smart Contract configurations, but also configurations for things like the blockchain client in use. This sometimes caused confusion for our users as there weren't sure where certain configurations should go.
```

Comment:
```suggestion
Embark's Smart Contract configuration has been highly declarative from day one. Configuring different deployment options and settings for various scenarios is a largely descriptive process when using Embark. However, we felt there was still room for improvement. Especially because Embark handles not only Smart Contract configurations, but also configurations for elements such as the user's choice of blockchain client. This sometimes caused confusion for our users since they weren't sure where certain configurations should go.
```